### ISSUE TITLE
Provide cross-validation functionality

### ISSUE THREAD

**Issue by arminwitte** [arminwitte/llsi#7]
In addition to the simulate method, a validate method would be nice to check against real output data with a test set (or the training set for that matter). As a prerequisite fit metrics like NRMSE-fit have to be implemented.
